excon.gemspec

```diff
@@ -26,4 +26,13 @@ Gem::Specification.new do |s|
   s.add_development_dependency('sinatra-contrib')
   s.add_development_dependency('json', '>= 1.8.5')
   s.add_development_dependency('puma')
+
+  spec.metadata = {
```

Comment:
Sounds great. I think this might actually need to be `s.metadata` instead of `spec.metadata` though? (in CI it looks like it is failing on spec being undefined). If you can make that update, I think it should be good to go in terms of merging. Thanks!

Reply:
Oops, fixed.

Reply:
No worries, easy mistake, thanks for the quick fix!